index.html

```diff
@@ -98,8 +104,10 @@ <h2 class="card-name">Tonic</h2>
       </ul>
       <button class="sec-button">see project</button>
     </div>
+    </div>
     <div class="card">
-      <img class="pic" src="img/Second pic.png" alt="second-img" />
+      <img class="pic pic2" src="img/desktop/Snapshoot Portfolio2.png" alt="second-img" />
+      <div class="Desktop-common">
```

Comment:
I can't find where this div is closed. Please try to avoid adding redundant code, and keep your project as clean as possible.